packages/story-editor/src/defaultConfig.js

```diff
@@ -110,7 +110,6 @@ const defaultConfig = {
   },
   isRTL: false,
   storyId: 1,
-  // @todo https://github.com/google/web-stories-wp/pull/9569#discussion_r739066940
   dashboardSettingsLink: '',
```

Comment:
@swissspidy Can you elaborate more on what you want to do with this `dashboardSettingsLink`.  `PublishPanel` is moved and `VideoOptimizationCheckbox` is being assembled with other checks in `wp-story-editor`

https://github.com/google/web-stories-wp/pull/9569#discussion_r739066940

Reply:
That's a good question 🙂 

`VideoOptimizationCheckbox` is just a way to tell the user they need to enable a setting so that they can optimize videos. Some platforms might not even have a setting for that (e.g. in a self-hosted playground it could always be enabled).

So I think it's safe to just move this component to `wp-story-editor` entirely.

Reply:
Moved in https://github.com/google/web-stories-wp/pull/9710/commits/fe65b43573491cf9783cbcbe481b329e78cba31d

Reply:
So can we now remove `dashboardSettingsLink` from `defaultConfig`?